app/views/admin/merchants/index.html.erb

```diff
@@ -1,9 +1,27 @@
 <h1>Merchants</h1>
 
-<ul>
-  <% @merchants.each do |merchant| %>
-    <li>
-      <%= link_to merchant.name, "/admin/merchants/#{merchant.id}" %>
+<h3>Enabled Merchants</h3>
+<ul id="enabled_merchants">
+  <% @merchants.enabled.each_with_index do |merchant, index| %>
+    <li id="merchant_<%=index%>">
+      <%= link_to "#{merchant.name}", "/admin/merchants/#{merchant.id}"  %>
+        <%= form_with url: "/admin/merchants/#{merchant.id}", method: :patch, local: true do |f| %>
+          <%= f.submit "Disable" %>
+          <%= f.hidden_field :disable, value: 'true' %>
+        <% end %>
     </li>
   <% end %>
-</ul>
\ No newline at end of file
+</ul>
+
+<h3>Disabled Merchants</h3>
+<ul id="disabled_merchants">
+  <% @merchants.disabled.each_with_index do |merchant, index| %>
+    <li id="merchant_<%=index%>">
```

Comment:
I think if we change it to id="merchant_#{merchant.id}" we could have CSS ids that match up with specific merchants for testing